test/compress/reduce_vars.js

```diff
@@ -53,9 +53,7 @@ reduce_vars: {
             console.log(-3);
             eval("console.log(a);");
         })(eval);
-        (function() {
-            return "yes";
-        })();
+        "yes";
```

Comment:
AFAICT all the `MAP` stuff has been around forever and hasn't changed.

Just wasn't used much due to others being unfamiliar with it, I suppose?